serverless/pages/spaces.mdx

```diff
@@ -0,0 +1,55 @@
+---
+slug: /serverless/spaces
+title: Spaces
+description: Organize your project and objects into multiple spaces
+---
+
+Spaces enable you to organize your dashboards and other saved
+objects into meaningful categories. Once inside a space, you see only
+the dashboards and saved objects that belong to that space.
+
+When you create and enter a new project, you're using the default space of that project.
+
+You can identify the space you're in or switch to a different space from the header.
+
+<DocImage size="l" url="../images/space-breadcrumb.png" alt="Space breadcrumb" />
+
+You can view and manage the spaces of a project from the **Spaces** page in **Management**.
+
+## Required permissions
+
+You must have the admin role on the project to manage its **Spaces**.
+
+## Create or edit a space
+
+You can create as many spaces as you need. 
```

Comment:
Technically the number of spaces is limited to 100 at this time, which includes the default space.

Reply:
Oh good to know! Didn't know we had this limitation. Thanks